curricula/factories.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth.models import User
+
+from factory import Sequence, PostGenerationMethodCall, SubFactory
+from factory.django import DjangoModelFactory
+
+from curricula.models import Curriculum, Unit
+
+
+class UserFactory(DjangoModelFactory):
+    class Meta:
+        model = User
+
+    username = Sequence(lambda n: "user_%d" % n)
+    password = PostGenerationMethodCall('set_password', 'password')
```

Comment:
borrowed from https://stackoverflow.com/questions/15616277/how-can-you-create-an-admin-user-with-factory-boy/17980414#17980414